server/routes/__test__/index.unit.test.js

```diff
@@ -6,7 +6,7 @@ describe("Test the root path  '/'  ", () => {
   test('The GET method ', (done) => {
     request(app)
       .get('/')
-      .expect('Content-Type', 'text/html; charset=utf-8')
+      .expect('Content-Type', 'text/html; charset=UTF-8')
```

Comment:
webpack produces a `index.html` with `charset=UTF-8' instead of `charset=utf-8`. Should this test be written to account for both cases?